src/table/types.ts

```diff
@@ -14,6 +14,7 @@ import {
   TableLayout,
   TotalsPosition,
 } from '../types';
+/* eslint-disable-next-line import/no-cycle */
 import { TSelectionActions } from './utils/selections-utils';
```

Comment:
Please fix it when you have time

Reply:
no, there is no problem when compiling and after compiling. It is only an eslint issue.